### ISSUE TITLE
 I don't want to use the CGO.

### ISSUE THREAD

**Issue by insionng** [glycerine/zygomys#12]
go get -u github.com/glycerine/zygomys/cmd/zygo

```
package github.com/glycerine/tmframe/vendor/github.com/codahale/blake2: C source files not allowed when not using cgo or SWIG: blake2b.c

```

**Comment by insionng** [comment]
Recommend the use of "github.com/dchest/blake2b"

**Comment by glycerine** [comment]
fixed with 23bc74770c945f51a2a1ea4cfc9823cca7d0ff5e
